`make env` -> `pip: No such file or directory`

```
$ make env
"/Users/whit537/bin/python2.7"  "./vendor/virtualenv-1.9.1.py"\
                                --unzip-setuptools \
                                --prompt="[gittip] " \
                                --never-download \
                                --extra-search-dir=./vendor/ \
                                --distribute \
                                ./env/
New python executable in ./env/bin/python2.7
Also creating executable in ./env/bin/python
Installing distribute.............................................................................................................................................................................done.
Installing pip......................done.
pip install -r requirements.txt
make: pip: No such file or directory
make: *** [env] Error 1
$
```

--- a/Makefile
+++ b/Makefile
@@ -9,7 +9,7 @@ test_env_files := defaults.env,tests/test.env,tests/local.env
 pip := $(env_bin)/pip
 honcho := $(env_bin)/honcho
 honcho_run := $(honcho) -e defaults.env,local.env run
-py_test := $(honcho) -e $(test_env_files) run py.test
+py_test := $(honcho) -e $(test_env_files) run $(env_bin)/py.test
 
 env: requirements.txt requirements_tests.txt setup.py
 	$(python) $(venv) \

